### ISSUE TITLE
Figure out docker image signing for Red Hat

### ISSUE THREAD

**Issue by infinitewarp** [cloudigrade/shiftigrade#18]
Before we can really "productize" cloudigrade at Red Hat, the build process needs to include signing of our docker images using keys that are trusted by Red Hat.

Some questions to answer here:

- Do we need a downstream fork specifically for Red Hat to build and release?
- If so, where does that fork live? How does it get deployed? Do we stop deploying the upstream cloudigrade?
- If not, how do we get cloudigrade things signed? In what repo do we save those images?
- When do we sign? For which commits, and at what point in the process?

Borja Tarraso (@btarraso) works in the Red Hat Product Security group and can be our point of contact as we sort through these things.

**Comment by infinitewarp** [comment]
This is no longer relevant since images are processed through Quay.
